frontend/src/scenes/session-recordings/player/rrweb/canvas/canvas-plugin.ts

```diff
@@ -88,7 +88,7 @@ export const CanvasReplayerPlugin = (events: eventWithTime[]): ReplayPlugin => {
 
                 const img = containers.get(e.data.id)
                 if (img) {
-                    img.src = target.toDataURL()
+                    img.src = target.toDataURL('image/jpeg', 0.6)
```

Comment:
on Mobile, I use 30% which is pretty good already, maybe we can reduce even more in the future if we think its still not great

Reply:
I only chose 0.6 to match what was on the backend during capture. That was also totally randomly picked so we should certainly revisit and reduce it in time